### DISCUSSION TITLE
Automatic Unstable Bar :O

### DISCUSSION THREAD

**Discussion by Neffohcs** [ppy/osu#21019] [Ideas]
Hi :D, first sorry my english i'm using translator :c
And I don't know if this is where I should send a suggestion.

I would like to suggest something for osu!lazer, a kind of automatic offset that adjusts itself at the end of each match, analyzing the unstablerate and considering the average player timing as "middle of UR bar".

So if all clicks were pressed to -50ms for example, it would still be an SS, because the play was consistent!

I think you got my idea, thanks ^-^

**Comment by Kaerulans** [comment]
The feature to calibrate the offset based on the last play already exists in lazer:
![Screenshot from 2022-10-30 14-23-28](https://user-images.githubusercontent.com/55179197/198875935-0693e17e-b7da-4c80-a9a7-ea5afffa0e04.png)

A play definitely shouldn't turn from a D to an SS just because your hits were consistent. Just play a map a couple of times to calibrate the offset and you should be good.

**Reply by Neffohcs** [reply]
yeah for me it's so frustrating I have many plays with good acc but 1miss, and when i get fc, it's with low acc
It's been 2 years and I can't fix it, but it seems like it's just me

![image](https://user-images.githubusercontent.com/117013607/198876919-26564f49-01d4-4688-aa54-2c0f8df1e57a.png)
the entire map is like this for me :c

**Reply by Neffohcs** [reply]
interesting is when I switch from optimal fps to vsync it generates input lag and helps in acc, but worsens the aim

**Reply by Neffohcs** [reply]
is there anything i can do any tips or "just play more"?

**Reply by Kaerulans** [reply]
This is a common problem. 
First, make sure your global offset is correct by calibrating it on some easier maps. Then try to concentrate on your rhythm sense and tapping without worrying that much about your aim. After a while, getting good acc should become automatic and you can again focus on aim.

**Reply by Neffohcs** [reply]
oh I will try, thank you ^-^
